### PR TITLE
fix: EventEmitter.listeners() should not throw on unknown listeners

### LINKED ISSUE #103: [Regression]: EventEmitter.listeners("foo") throws if no listener exists
The following:

```py
from pyee import EventEmitter
ee = EventEmitter()
ee.listeners("foo")
```

results in:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/.../env/lib/python3.10/site-packages/pyee/base.py", line 229, in listeners
    return list(self._events[event].keys())
KeyError: 'foo'
>>> 
```

since version 9.0.0. In version `<8` it was returning an empty list instead. This most likely broke in #99 when the `defaultdict` of `EventEmitter._events` got changed to a normal `dict`.

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes #103

### CODE REVIEW

**Comment by jfhbrook** [comment]
Oh, you got it for me! Cool!

File: tests/test_sync.py [hunk 1]
```diff
@@ -240,6 +240,13 @@ def once_handler():
     call_me.assert_not_called()
 
 
+def test_listeners_does_work_with_unknown_listeners():
```
Comment (jfhbrook) [inline tests/test_sync.py:243]: I didn't even have to ask for a test. Beautiful!

**Review by jfhbrook — APPROVED** [review]
This looks great! I'll pull the release levers shortly! Thanks so much!!

**Comment by mxschmitt** [comment]
@jfhbrook thank you! We use it very heavily at [Playwright](https://github.com/microsoft/playwright-python) and it works like a charm! 💯

**Pull request merged**